File: check_funciones.php
<?php
$serverName = "srvflorlab";
$database = "EnterpriseTest";
$uid = "sa";
$pwd = "@enterprise123456";

try {
    $conn = new PDO("sqlsrv:server=$serverName;Database=$database;TrustServerCertificate=true", $uid, $pwd);
    $conn->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
    
    $stmt = $conn->query("SELECT ROUTINE_NAME, ROUTINE_TYPE FROM INFORMATION_SCHEMA.ROUTINES WHERE ROUTINE_NAME LIKE '%crypt%' OR ROUTINE_NAME LIKE '%des%' OR ROUTINE_NAME LIKE '%enc%'");
    $funciones = $stmt->fetchAll(PDO::FETCH_ASSOC);
    print_r($funciones);
} catch(Exception $e) {
    die("Error de conexión: " . $e->getMessage());
}

File: check_funciones.cs
using System;
using System.Data.SqlClient;

class Program
{
    static void Main()
    {
        string connStr = "Server=srvflorlab;Database=EnterpriseTest;User Id=sa;Password=@enterprise123456;";
        try {
            using (SqlConnection conn = new SqlConnection(connStr)) {
                conn.Open();
                SqlCommand cmd = new SqlCommand("SELECT ROUTINE_NAME FROM INFORMATION_SCHEMA.ROUTINES WHERE ROUTINE_TYPE='FUNCTION'", conn);
                using (SqlDataReader reader = cmd.ExecuteReader()) {
                    while (reader.Read()) {
                        Console.WriteLine(reader["ROUTINE_NAME"].ToString());
                    }
                }
            }
        } catch (Exception ex) {
            Console.WriteLine(ex.Message);
        }
    }
}
